Userland/Libraries/LibWeb/CSS/Length.cpp

```diff
@@ -19,14 +19,14 @@
 
 namespace Web::CSS {
 
-Length::FontMetrics::FontMetrics(CSSPixels font_size, Gfx::FontPixelMetrics const& pixel_metrics, CSSPixels line_height)
+Length::FontMetrics::FontMetrics(CSSPixels font_size, Gfx::FontPixelMetrics const& pixel_metrics)
     : font_size(font_size)
     , x_height(pixel_metrics.x_height)
     // FIXME: This is only approximately the cap height. The spec suggests measuring the "O" glyph:
     //        https://www.w3.org/TR/css-values-4/#cap
     , cap_height(pixel_metrics.ascent)
     , zero_advance(pixel_metrics.advance_of_ascii_zero + pixel_metrics.glyph_spacing)
-    , line_height(line_height)
+    , line_height(round(pixel_metrics.line_spacing()))
```

Comment:
`CSSPixels::nearest_value_for()` instead of `round()`.

Reply:
I specifically want to round to an integer here, as that is what other engines do.

Reply:
Note: `CSSPixels` has it's own round overload

Reply:
Are you suggesting that this should be `round(CSSPixels(pixel_metrics.line_spacing()))` instead of `CSSPixels(round(pixel_metrics.line_spacing()))`?
If so, what is the difference / in what situation does it avoid a bug? :thinking:

Reply:
I forgot CSSPixels aren't integers. :man_facepalming: